### ISSUE TITLE
Give elements a clear focus outline

### ISSUE THREAD

**Issue by dhinrichs-scottlogic** [freeCodeCamp/design-style-guide#47]
Hi, I hope it's okay to raise an issue here. I asked on the contributor forum if this page can be worked on, and it was recommended to open an issue to suggest improvements. Let me know what you think :) 

**Description**
The Design Style Guide webpage currently does not meet accessibility standards. To ensure freeCodeCamp can be enjoyed by all, I would like to work on the following improvement:

**Focus visibility for easy keyboard navigation.** 
WCAG 2.4.7 Focus Visible
https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

**Reproduction steps**
Steps to reproduce the behaviour:
1. Go to https://design-style-guide.freecodecamp.org/
2. Use the Tab Key to navigate through all interactive elements
3. See the lack of outline 

**Expected behaviour**
Each focusable element shows a clear outline when in focus. A clear outline is defined as at least 2px thick and with at least 3:1 contrast ratio: https://w3c.github.io/wcag/understanding/focus-visible-enhanced.html

**Suggested Design**
A 3px white solid outline with a 3px outline-offset as can be seen in the screenshot below.

**Screenshot of suggested focus indication**
![image](https://github.com/freeCodeCamp/design-style-guide/assets/125262707/f497b89a-593a-41a9-8468-ee249d9cf035)

**Acceptance criteria**
**Given** A user navigates the web using a keyboard
**When** they read through the freeCodeCamp Design Style Guide
**Then** the clear focus indication lets them know which element they are currently on.

**Comment by huyenltnguyen** [comment]
Hey @dhinrichs-scottlogic, thank you for reporting the issue and for giving this page some care :)

> I hope it's okay to raise an issue here.

Usually we expect issues to be created in the main repo (https://github.com/freeCodeCamp/freeCodeCamp) since all of the maintainers are watching/monitoring it, and there is less chance for the issues to be missed. But I guess creating issues here is also fine.

---

I agree that the focus outline of interactive elements on this page needs to be improved. I'd suggest going with a blue border instead of white, since this is the style we have been using in the main codebase.

Here is how the focus outline is implemented in the main repo: https://github.com/freeCodeCamp/freeCodeCamp/blob/3a93394a2f2655e0388d1d62edee48b4b5165e42/client/src/components/layouts/global.css#L201-L204.

Here is how it looks on https://www.freecodecamp.org/:

<img width="184" alt="Screenshot 2024-01-03 at 17 36 52" src="https://github.com/freeCodeCamp/design-style-guide/assets/25715018/48304fd9-41de-41b1-a40d-2772a1c5c556">

---

You're welcome to create a PR for the fix.

Usually we ask contributors to follow our contributing guide: https://contribute.freecodecamp.org, but it's is a bit different here since the repo setup and testing aren't that involved, compared to the main repo. For this case, the one guide we recommend you going through would be https://contribute.freecodecamp.org/#/how-to-open-a-pull-request. Also, you can join us in [our chat room](https://discord.gg/PRyKn3Vbay) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing; the moderator team can guide you through this.

**Comment by dhinrichs-scottlogic** [comment]
Thank you for the quick reply @huyenltnguyen
I didn't consider creating an issue in the main repo, but if that's being watched more frequently, I'll do that in the future for other issues. 
I'll use the focus outline you suggested: The blue border like in the main codebase. 
I'll make the changes and then create a PR following the freeCodeCamp guidelines. Thanks again :)
